sharq_project/controllers/__init__.py

```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 
-from . import controllers
\ No newline at end of file
+from . import controllers
```

Comment:
@Buniadi remove the controllers from here and also delete the folder by the name of controllers